cmd/snap-bootstrap/bootstrap/bootstrap.go

```diff
@@ -223,6 +217,15 @@ func tpmSealKey(key partition.EncryptionKey, rkey partition.RecoveryKey, options
 			return err
 		}
 	}
+
+	// TODO:UC20: get cmdline definition from bootloaders
+	kernelCmdlines := []string{
+		// run mode
+		"snapd_recovery_mode=run console=ttyS0 console=tty1 panic=-1",
+		// recover mode
+		fmt.Sprintf("snapd_recovery_mode=run snapd_recovery_system=%s console=ttyS0 console=tty1 panic=-1", options.SystemLabel),
```

Comment:
this is wrong, the snapd_recovery_mode should be `recover`, not `run` here.

```suggestion
		fmt.Sprintf("snapd_recovery_mode=recover snapd_recovery_system=%s console=ttyS0 console=tty1 panic=-1", options.SystemLabel),
```

Reply:
Ah ugh, I focused too much in the snapd_recovery_system part and didn't replace the main mode. Sorry!